src/client/Atmosphere.js

```diff
@@ -152,6 +152,7 @@ export default class Atmosphere extends Environment {
       }
     });
     this.emitSubscribe(text, variables, opId);
+    // return {dispose: () => this.safeSocketUnsubscribe(subKey)};
```

Comment:
relay recommends that you write your socket handler to return a `disposable`. Then, you can call those disposables when needed. I structured it a bit differently to make more sense with our custom QueryRenderer, but I wanted to leave this in here in case I wanted to switch back to the suggested use.